documents/validator.go

```diff
@@ -264,5 +277,6 @@ func SignatureValidator(idService identity.ServiceDID) ValidatorGroup {
 		baseValidator(),
 		signingRootValidator(),
 		signaturesValidator(idService),
+		//transition_validator
```

Comment:
Let us not add the integration here. We will add the validator as part of different PR